pkg/gateway/multiparts/tracker.go

```diff
@@ -5,14 +5,25 @@ import (
 	"database/sql/driver"
 	"encoding/json"
 	"errors"
+	"fmt"
 	"strings"
 	"time"
 
-	"github.com/treeverse/lakefs/pkg/db"
+	kv "github.com/treeverse/lakefs/pkg/kv"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 type Metadata map[string]string
 
+var ErrAlreadyExists = errors.New("upload ID already exists")
+
+// MultipartUpload data structure
+// UploadID A unique identifier for the uploaded part
+// Path Multipart path in repository
+// CreationDate Creation date of the part
+// PhysicalAddress Physical address of the part in the storage
+// Metadata Additional metadata as required (by storage vendor etc.)
+// ContentType Original file's content-type
 type MultipartUpload struct {
 	UploadID        string    `db:"upload_id"`
 	Path            string    `db:"path"`
```

Comment:
We usually document fields inline

```suggestion
// MultipartUpload represents an ongoing multipart upload to the blockstore
type MultipartUpload struct {
	// UploadID A unique identifier for the uploaded part
	UploadID        string    `db:"upload_id"`
	
	// Path Multipart path in repository
	Path            string    `db:"path"`
	
	...
```